docs/org/reference/language/modules.org

```diff
@@ -13,15 +12,16 @@ module ModuleName;
 end;
 #+end_example
 
-A _Juvix project_ is a collection of Juvix modules inside one main project
-folder containing a metadata file named =juvix.yaml=. Each Juvix file has to
-define a _module_ of the same name. The name of the module must coincide with
-the path of the its file respect to its project. For example, if the file is
-=root/Data/List.juvix= then the module must be called =Data.List=, assuming
-=root= is the project's folder.
+A _Juvix project_ is a collection of Juvix modules inside one main
```

Comment:
I cannot see the difference here. Did you just format the text with less width I guess?

Reply:
Mostly only formatting, but I also changed the language slightly.